### DISCUSSION TITLE
Deployr/Redeploy - An unexpected internal error occurred

### DISCUSSION THREAD

**Discussion by ttsan-nooz** [vercel/community#4201] [Help]
### Summary

Hello,

I'm looking for some help.
Since yesterday, I've issue trying to deploy/redeploy my project.
When I push new commits, it didn't trigger any deploy and when I'm forcing a redeploy I've the error:
An unexpected internal error occurred.

Thank you

### Steps to Reproduce

Click on 'Redeploy'

**Comment by yasielhp** [comment]
I have had the same error since yesterday, and today is still the same, I have tried several methods and it keeps happening, it is as if the Vercel autodeployment is not done when changes are made in the repository.

**Reply by yasielhp** [reply]
I don't know if Vercel has a new limit on automatic deployments?

**Comment by amyegan** [comment]
Hobby accounts are currently limited to 100 deployments created per day: https://vercel.com/docs/limits/overview#general-limits

Certain project settings may also block automatic deployment. If you don't seem to be running into the build/deployment limit, the next things to check are [Ignored Build Step](https://vercel.com/docs/projects/overview#ignored-build-step) and [git.deploymentEnabled](https://vercel.com/docs/projects/project-configuration/git-configuration#git.deploymentenabled) settings.
